### PR TITLE
Fix bugs, Migrate to Material, Refactor code of Admin Components

### PR DESCRIPTION
**Pull request by fragm3**
Fixes Partially #1996, #1997

Changes: 
 - Migrated Tabs to Material UI
 - Refactored Code
 - Fixed skill name not appearing in Skills tab
 - Fixed if user/admin directly access an inner route like `/admin/logs`, earlier it was showing Not found page, now with the newer approach, it is fixed.
 - Fixed Skill Logs to fetch response
 - Created common components.

Note: More iterations required. Table component still needs to be migrated. ListUser is unable to fetch any data, fix that

Surge Deployment Link: https://pr-2001-1-fossasia-susi-skill-cms.surge.sh

### CODE REVIEW

**Comment by fragm3** [comment]
@akshatnitd @mariobehling, please review

**Comment by harsh-jindal** [comment]
@fragm3 Rest is fine. Will have to work on Mobile views. I'll work on it after this gets merged.

**Comment by fragm3** [comment]
@harsh-jindal I am not able to find what is causing those 2 icons. I looked into antd docs, couldn't find.

About the filter, its a design principle used by antd to keep filter right aligned and sort left aligned. No worries about it

**Comment by harsh-jindal** [comment]
@fragm3 I guess the double icons are appearing because for importing table older style of antd is being used.
About the design principles by antd I still prefer to keep the same UI as before. Because keeping the sort icon right aligned, in our case appears to be far from the text and it looks out of place.

Also can we same navbar as before because this one has issue in mobile view. I cannot scroll to system logs.
Screenshot 2019-05-29 at 9 19 21 AM

Also @akshatnitd can tell more on whether to change UI or not.

File: src/components/Admin/SystemLogs/SystemLogs.js [hunk 1]
```diff
@@ -40,15 +34,26 @@ class SystemLogs extends React.Component {
   }
 
   loadSystemLogs = count => {
-    fetchSystemLogs({ count })
-      .then(payload => {
+    const { accessToken } = this.props;
+    axios
```
Comment (harsh-jindal) [inline src/components/Admin/SystemLogs/SystemLogs.js]: Why not put this in api folder as we have all the api's there it self

**Comment by fragm3** [comment]
@harsh-jindal I had discussed this with Akshat sir beforehand, I agree with the UI of antd looks cleaner, but we still shift to material-ui. 

@akshatnitd please elaborate.

File: src/components/Admin/SystemLogs/SystemLogs.js (hunk 1 above)
Comment (fragm3) [reply src/components/Admin/SystemLogs/SystemLogs.js]: It is primarily a problem of axios, I saw an unsolved issue of text type responses as well, so we will have to deal with this in the file itself.

**Comment by fragm3** [comment]
@harsh-jindal fixed the double table pagination

File: src/components/Admin/SystemLogs/SystemLogs.js (hunk 1 above)
Comment (harsh-jindal) [reply src/components/Admin/SystemLogs/SystemLogs.js]: Okay

**Comment by harsh-jindal** [comment]
@fragm3 If this UI has been suggested by @akshatnitd then okay.

**Comment by fragm3** [comment]
@akshatnitd made the changes

**Comment by fragm3** [comment]
@akshatnitd made the changes

File: src/components/Admin/Admin.js [hunk 2]
```diff
@@ -54,358 +50,115 @@ class Admin extends Component {
...
+      case 'settings':
+        heading = 'System Settings';
+        value = 3;
+        break;
+      case 'logs':
+        heading = 'System Logs';
+        value = 4;
+        break;
+      default:
+        return;
```
Comment (akshatnitd) [inline src/components/Admin/Admin.js]: Remove return, set heading='Admin' and value=0 here

File: src/components/Admin/Admin.js [hunk 3]
```diff
@@ -54,358 +50,115 @@ class Admin extends Component {
...
   };
 
-  handleTabChange = activeKey => {
-    if (activeKey === '2') {
-      this.props.history.push('/admin/users');
+  handleTabChange = (event, value) => {
+    this.setState({ value });
+    switch (value) {
+      case 0:
+        this.props.history.push('/admin');
```
Comment (akshatnitd) [inline src/components/Admin/Admin.js]: Use `history.replace` instead

**Comment by fragm3** [comment]
@akshatnitd made the changes

**Pull request merged**